webapp/src/components/pages/Home.vue

```diff
@@ -0,0 +1,271 @@
+<template>
+  <div
+    class="mx-0 lg:mx-8 text-slate-600 dark:text-slate-300"
+  >
+
+  <!-- First screen -->
+    <div class="mb-20 lg:mb-0 lg:h-dvh flex flex-col justify-around pb-12">
+  <!-- Disco logo -->
+    <div class="flex flex-col justify-center items-center space-y-4">
+        <div class="">
+          <DiscoGIF v-if="themeStore.currentTheme == 'light'" class="pb-4"/>
+          <DiscoLogo v-else class="w-[640px] h-[264px]"/>
+        </div>
+        <span
+          class="text-xl sm:text-2xl md:text-3xl lg:text-4xl text-slate-600 dark:text-slate-300"
+        >
+          <span class="font-disco text-disco-cyan font-semibold">DIS</span
+          >tributed
+          <span class="font-disco text-disco-blue font-semibold">CO</span
+          >llaborative Learning
+        </span>
+      </div>
+      <div class="text-center md:text-left mt-24">
+        <p
+          class="text-3xl sm:text-4xl md:text-5xl lg:text-6xl xl:text-7xl font-semibold
+          text-slate-700 dark:text-white leading-9"
+        >
+          Train AI Models Together.<br/>
+          Keep Data Private.
+        </p>
+        <p class="text-base sm:text-base md:text-lg lg:text-xl
+          text-slate-600 dark:text-slate-300 mt-3"
+        >
+          Build and train AI models without sharing any data. <br/>
+          Machine Learning directly in your browser.
+        </p>
+        <div class="flex justify-center mb-12">
+          <button
+            type="button"
+            @click="goToTaskList()"
+            class="
+            mt-4 px-6 py-3 min-w-[8rem]
+            text-xl text-white tracking-wide
+            bg-orange-400 rounded-xl duration-200
+            hover:scale-105 transform
+            drop-shadow-[0px_0px_5px_rgba(251,146,60,0.5)]
+            hover:drop-shadow-[0px_0px_10px_rgba(251,146,60,0.5)]"
+          >
+            Start training
+          </button>
```

Comment:
better to use a `RouterLink`

```suggestion
          <RouterLink
            to="/list"
            class="
            mt-4 px-6 py-3 min-w-[8rem]
            text-xl text-white tracking-wide
            bg-orange-400 rounded-xl duration-200
            hover:scale-105 transform
            drop-shadow-[0px_0px_5px_rgba(251,146,60,0.5)]
            hover:drop-shadow-[0px_0px_10px_rgba(251,146,60,0.5)]"
          >
            Start training
          </RouterLink>
```